docs/notebooks/Kusto-Analysis.ipynb

```diff
@@ -0,0 +1,1642 @@
+{
```

Comment:
<p>You could also just using import msticpy and then msticpy.init_notebook.</p><p>This imports QueryProvider and you can access refresh_config from msticpy.settings.refresh_config()</p><p>This isn't saving you much to be honest but it might lessen the pain one day when we move things around.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/msticpy/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Sounds sensible. Now I know about init_notebook! :)</p>